dags/synapse-by-the-numbers-dag.py

```diff
@@ -22,7 +22,7 @@
 
 dag_config = {
     # run on the 1st of every month at midnight
```

Comment:
nit: Update comment to reflect this runs the 2nd

Reply:
Whoops, thanks for catching that!